.github/workflows/build-windows.yml

```diff
@@ -104,7 +103,7 @@ jobs:
     - name: Build exe
       run: python distro.py build_exe
     - name: Copy Tktable2.11
-      run: xcopy libs\win64\Tktable2.11 build\exe.win-amd64-${{ matrix.python-version }}\lib\tkinter\tk8.6 /y
+      run: xcopy libs\win64\Tktable2.11 build\exe.win-amd64-${{ matrix.python-version }}\lib\tcltk\tk8.6 /y
```

Comment:
Matches the tkinter target directory [hook change](https://github.com/marcelotduarte/cx_Freeze/pull/1526/files) in cx-Freeze 6.12